.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+minimum_pre_commit_version: "2.9.0"
```

Comment:
Why is this needed?

Reply:
This is similar to the need of pinning versions in requirements.txt. If we allow having the latest version, then we can't guarantee reproducibility. In case of pre-commit it is even better - you just run:
pre-commit auto-update and these versions are updated automatically